### DISCUSSION TITLE
Hello  CRON is running too long, check if mail system works correctly.

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#15283] [General] [answered]
Yeti sending me emails: 
Hello
CRON is running too long, check if mail system works correctly.

But from what I see cron running smoothly. Any clues?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
turn on cron logs, and see how much cron is running?
In the new version, cron has been optimized so it will work better

https://doc.yetiforce.com/code/classes/Config-Debug.html#property_DEBUG_CRON
